tests/unit/resizable/core.js

```diff
@@ -244,4 +244,57 @@ QUnit.test( "nested resizable", function( assert ) {
 	outer.remove();
 } );
 
+QUnit.test( "Resizable with scrollbars and box-sizing: border-box", function( assert ) {
+	assert.expect( 2 );
+
+	var elementContent = $( "<div>" )
+			.css( {
+				width: 50,
+				height: 200,
+				padding: 10,
```

Comment:
Can you also set some margin to make sure that doesn't interfere?

Also, provide explicit values with the `px` suffix, not numbers. Passing numbers & depending on them getting `px` auto-appended is deprecated and disabled for most CSS properties in jQuery 4.0.0.

The properties here work (see the regex at https://github.com/jquery/jquery/blob/4.0.0-beta.2/src/css/isAutoPx.js) but this is mostly for backwards compat reasons, let's not depend on that here.